src/main/java/athena/timetable/TimetableDay.java

```diff
@@ -0,0 +1,31 @@
+package athena.timetable;
+
+import athena.tasklist.TaskList;
+
+import java.util.Date;
+
+/**
+ * Represents a day in the timetable.
+ * Contains the date and the list of tasks, which is a subset of all the tasks
+ * that the user registered into the program.
+ */
+public class TimetableDay {
+    private Date date;
+    private TaskList taskList;
+
+    public TimetableDay(Date date, TaskList taskList) {
+
+    }
+
+    /**
+     * Generates a string containing the date and list of tasks that is to
+     * be printed to the user.
+     *
+     * @return A string containing the date and list of tasks.
+     */
+    @Override
+    public String toString() {
+        // TODO: Generate a string containing the date and the tasks
+        return "";
+    }
+}
```

Comment:
@alstontham I think it would be good to overload the `toString` method so that we can print the object directly. You can work on printing out a `Date` and `TaskList` on your side first then we can move that code here later. 

Or you can fetch this branch and modify it, then make a pull request to my branch.

Reply:
> @alstontham The timetable given to the `Ui` will be a list of `TimetableDay` objects, each containing a date and a list of `Task`s. So you can work on printing that now.

Ok! Will work on this

Reply:
> @alstontham I think it would be good to overload the `toString` method so that we can print the object directly. You can work on printing out a `Date` and `TaskList` on your side first then we can move that code here later.
> 
> Or you can fetch this branch and modify it, then make a pull request to my branch.

Sure, will see what I can do